app/dashboard/vite.config.ts

```diff
@@ -21,10 +21,14 @@ await appConfig.readEnvironmentFromFile()
 export default vite.defineConfig({
   server: { port: SERVER_PORT, headers: Object.fromEntries(common.COOP_COEP_CORP_HEADERS) },
   plugins: [
+    // @ts-expect-error It seems like there is a type error caused by differences in type
+    // definitions between Vite versions.
```

Comment:
This version mismatch is now fixed in develop, those expect-errors will probably be flagged after merge.